lib/Interpreter/Interpreter.cpp

```diff
@@ -265,6 +275,30 @@ namespace cling {
             } else
               cling::errs() << Sym << " not in Module!\n";
           }
+
+          const clang::Decl* Scope =
+              LangOpts.CPlusPlus
+                  ? m_LookupHelper->findScope("cling::runtime",
+                                              LookupHelper::NoDiagnostics)
+                  : CI->getSema().getASTContext().getTranslationUnitDecl();
+          if (!Scope) {
+            cling::errs() << "Scope for gCling was not found\n";
+          }
+          else if (const clang::ValueDecl* gCling =
+              m_LookupHelper->findDataMember(Scope, "gCling",
+                                             LookupHelper::NoDiagnostics)) {
+            using namespace utils;
+            const std::string Name = Analyze::maybeMangleDeclName(gCling);
+            if (!Name.empty()) {
+              // gCling gets linked to top-most Interpreter.
```

Comment:
My mistake, this was culled from a changes that moved to **thisCling** which links to the child as parent can always be reached through it.